Content.Shared/Shuttles/Systems/SharedShuttleSystem.cs

```diff
@@ -41,7 +41,7 @@ public override void Initialize()
     /// </summary>
     public bool CanFTLTo(EntityUid shuttleUid, MapId targetMap, EntityUid consoleUid)
     {
-        var mapUid = _mapManager.GetMapEntityId(targetMap);
+        var mapUid = Maps.GetMap(targetMap);
```

Comment:
```suggestion
        var mapUid = Maps.GetMapOrInvalid(targetMap);
```
this would be the replacement if you want the behaviour to be the same